src/Webserver.cc

```diff
@@ -121,16 +121,22 @@ bool Webserver::parse_config(const char* file_name) {
 }
 
 RequestHandler* Webserver::get_handler(std::string uri) {
-    size_t last = uri.find_last_of("/");
-    std::string prefix = uri.substr(0, last);
-    std::unordered_map<std::string, RequestHandler*>::const_iterator found = handler_map.find(prefix);
+    std::unordered_map<std::string, RequestHandler*>::const_iterator found = handler_map.find(uri);
```

Comment:
Rather than checking twice, what you could do is just check if the last character is a slash, and remove it if it is. (You can check the last character with the .back() function). Also, we still need to find the longest matching prefix. Right now this doesn't support subdirectories